examples/webgl_geometry_text.html

```diff
@@ -215,49 +165,56 @@
 				document.addEventListener( 'keypress', onDocumentKeyPress );
 				document.addEventListener( 'keydown', onDocumentKeyDown );
 
-				document.getElementById( 'color' ).addEventListener( 'click', function () {
+				//
 
-					pointLight.color.setHSL( Math.random(), 1, 0.5 );
-					hex = decimalToHex( pointLight.color.getHex() );
+				const params = {
+					changeColor: function () {
 
-					updatePermalink();
+						pointLight.color.setHSL( Math.random(), 1, 0.5 );
 
-				} );
+					},
+					changeFont: function () {
```

Comment:
Maybe use a list picker for font and a color picker for color?

Reply:
Didn't want to change the behavior of the picks. I think they're are fine.